modules/check_in/app/services/travel_claim/client.rb

```diff
@@ -63,11 +63,8 @@ def submit_claim(token:, patient_icn:, appointment_date:)
         req.body = claims_data.merge({ ClaimantID: patient_icn, Appointment:
           { AppointmentDateTime: appointment_date } }).to_json
       end
-    rescue Faraday::TimeoutError => e
-      log_exception_to_sentry(e,
-                              { uuid: check_in.uuid },
-                              { external_service: service_name, team: 'check-in' },
-                              :error)
+    rescue Faraday::TimeoutError
+      Rails.logger.error(message: 'BTSSS Timeout Error', uuid: check_in.uuid)
```

Comment:
the log entry needs to be hashed for DD to contextualize it: https://docs.datadoghq.com/logs/log_collection/ruby/?tab=lograge#best-practices
```
      logger.error({message: 'BTSSS Timeout Error', uuid: check_in.uuid})
```

Reply:
Logging entry to Rails.logger as `message: 'BTSSS Timeout Error', uuid: check_in.uuid`  gets treated as hash.  As we discussed, will push these changes & monitor in datadog.  If the log doens't get contextualized in datadog, will send the log as hash.